### PR TITLE
Fix ClassNotFoundException in kafka 3.0.0 and newer

### PR DESCRIPTION
**Pull request by ladislavmacoun**
Kafka removed `kafka.security.auth.SimpleAclAuthorizer` class in version
3.0.0[1] which leads to java.lang.ClassNotFoundException during the kafka server
startup.

This patch adds version checking and replaces
kafka.security.auth.SimpleAclAuthorizer with the
kafka.security.authorizer.AclAuthorizer for version 3.0.0 and newer.

[1] : https://github.com/apache/kafka/commit/976e78e405d57943b989ac487b7f49119b0f4af4#diff-e0ccf1b5c964d2c303b6a69a8b8b67df5a6bfbae8aa514f580d353c4c6bf8e36

Signed-off-by: Ladislav Macoun <ladislavmacoun@gmail.com>

### CODE REVIEW

File: trivup/apps/KafkaBrokerApp.py [hunk 1]
```diff
@@ -443,3 +443,11 @@ def deploy(self):
         # easily called.
         self.env_add('PATH', os.environ.get('PATH') + ':' +
                      os.path.join(destdir, 'bin'), append=False)
+
+    def _add_simple_authorizer(self, conf_blob):
+        # Kafka removed SimpleAclAuthorizer class in v3.0.0
+        # https://github.com/apache/kafka/commit/976e78e405d57943b989ac487b7f49119b0f4af4#diff-e0ccf1b5c964d2c303b6a69a8b8b67df5a6bfbae8aa514f580d353c4c6bf8e36
+        if self.version[0] >= 3:
+            conf_blob.append('authorizer.class.name=kafka.security.authorizer.AclAuthorizer')
```
Comment (edenhill) [inline trivup/apps/KafkaBrokerApp.py]: this probably needs a noqa E501 too

**Review by edenhill — CHANGES_REQUESTED** [review]
Great stuff, minor nit

File: trivup/apps/KafkaBrokerApp.py (hunk 1 above)
Comment (ladislavmacoun) [reply trivup/apps/KafkaBrokerApp.py]: Sorry, my bad, added to the commit.

**Review by edenhill — APPROVED** [review]
LGTM!

**Comment by edenhill** [comment]
Good work, thanks!

**Pull request merged**